Assets/Scripts/Interactables/InteractableUI.cs

```diff
@@ -90,8 +90,6 @@ private void SetInteractionTipText(Transform tipParent, InteractionOption intera
 
   private string KeyBindingText(InteractionType interactionType)
   {
-    if (interactionType == InteractionType.UseMouth) return interactionBindings["UseMouth"];
-    if (interactionType == InteractionType.UseBody) return interactionBindings["UseBody"];
-    return interactionBindings["UsePaw"];
+    return interactionType.ToString();
```

Comment:
This isn't perfect since the output will be missing a space, but we lost the reference to interaction bindings when I was cleaning up the player controller. Happy to patch this, but wanted to get the PR open as I was waffling. Let me know.